src/main/java/net/minecraftforge/client/event/ScreenEvent.java

```diff
@@ -1045,10 +1045,24 @@ public void setNewScreen(Screen newScreen)
      */
     public static class Closing extends ScreenEvent
     {
+        @Nullable
+        private final Screen newScreen;
+
         @ApiStatus.Internal
-        public Closing(Screen screen)
+        public Closing(Screen screen, @Nullable Screen newScreen)
         {
             super(screen);
+            this.newScreen = newScreen;
+        }
+
+        /**
+         * {@return The screen that will be opened after the current screen is closed.<br>
+         * If null then the game will return to the gameplay.}
```

Comment:
Not sure why this needs a line break. Also, the period at the end can be removed, it is added by the javadoc.

Reply:
Also, the sentence will become `Returns ...`, so remove the capitalization of `The screen`.

Reply:
Feels weird.
The build in Javadoc viewer (nice format) Formats it like this.
![image](https://user-images.githubusercontent.com/32039051/194936835-37d19641-2821-4f15-963f-201dd7a026da.png)
No appended period. 
It is also `Returns:` So not sure if you would capitalize after a colon.

Reply:
The period is added at the end of the method description, not the actual tag itself.

Reply:
Additionally, I agree with the capitalization removal, even if the javadoc viewer doesn't show it.